_config.yml

```diff
@@ -9,6 +9,9 @@ plugins:
 whitelist:
   - jekyll-redirect-from
 
+copy_to_destination:
+  - node_modules/identity-style-guide/dist/assets
+
```

Comment:
Unfortunately, for [Jekyll configuration](https://jekyllrb.com/docs/configuration/options/), the `exclude` filter [takes precedence over](https://github.com/jekyll/jekyll/issues/4791) the `include` filter. I’d rather specifically include this one folder, rather than including all of `node_modules` (excluded by default).

Copying a bunch of static files can seriously slow down the build process, so I think this is the right way to go (the custom plugin) — but let me know if you think it’s something worth revisiting, or if you know of a better way!